easybuild/easyconfigs/m/mDust/mDust-20231029-GCC-10.3.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'mDust'
```

Comment:
On the github page and in the source code they use `mdust` everywhere, so perhaps we should stick to that? This is also used in the description below.